components/landing/import-peerings.tf

```
@@ -0,0 +1,17 @@
+# import hub-to-ingest00-vnet-sbox peering
+import {
+  to = module.data_landing_zone["00"].module.vnet_peer_hub.azurerm_virtual_network_peering.target_to_initiator
+  id = "/subscriptions/df72bb30-d6fb-47bd-82ee-5eb87473ddb3/resourceGroups/ingest00-network-sbox/providers/Microsoft.Network/virtualNetworks/ingest00-vnet-sbox"
```

```suggestion
  id = "/subscriptions/df72bb30-d6fb-47bd-82ee-5eb87473ddb3/resourceGroups/ingest00-network-sbox/providers/Microsoft.Network/virtualNetworks/ingest00-vnet-sbox/"
```

Need /virtualNetworkPeerings/<nameofthepeering> appended to the end of id